core/src/main/java/nl/nn/adapterframework/align/Json2Xml.java

```diff
@@ -130,11 +141,13 @@ public void handleElementContents(XSElementDeclaration elementDeclaration, JsonV
 
 
 	@Override
-	public String getNodeText(XSElementDeclaration elementDeclaration, JsonValue node) {
+	public String getNodeText(XSElementDeclaration elementDeclaration, JsonValue node) throws SAXException {
 		String result;
 		if (node instanceof JsonString) {
-			result=((JsonString)node).getString();
-		} else if (node instanceof JsonStructure) { // this happens when override key is present without a value
+			result = ((JsonString) node).getString();
+		} else if (node instanceof JsonArray) {
+			throw new SAXException("Expected simple element, got instead an array-value: [" + node + "]");
+		} else if (node instanceof JsonStructure) { // this happens when override key is present without a value (?? Is that so?)
 			result=null;
 		} else {
```

Comment:
In master hadden we hier ook nog dit:
```
			if (!jsonObject.isEmpty()) {
				throw new SAXException("Expected simple element, got instead an object-value: [" + node + "]");
			}
			result=null;
```

Waarom hadden we die extra fix eigenlijk niet in 7.9 gezet? Weet jij dat nog?

Reply:
Ik vond dat nog een beetje eng...